spec/ParseGeoPoint.spec.js

```diff
@@ -69,17 +69,13 @@ describe('Parse.GeoPoint testing', () => {
   });
 
 
-  it('geo point exception two fields', (done) => {
+  it('geo point supports more than one field', (done) => {
     var point = new Parse.GeoPoint(20, 20);
     var obj = new TestObject();
     obj.set('locationOne', point);
     obj.set('locationTwo', point);
-    obj.save().then(() => {
-      fail('expected error');
-    }, (err) => {
-      equal(err.code, Parse.Error.INCORRECT_TYPE);
-      done();
-    });
+    obj.set('locationThree', point);
+    obj.save().then(done, done.fail);
```

Comment:
@dplewis This supports saving 3 points, so yay. But where is the test that ensures that retrieving them actually retrieves them correctly?

This provides two things: 1) actually ensuring that round trips work. 2) Giving someone a demonstration of how to retrieve them (ie, mini tutorial).

Reply:
Good point i'll add it now.